### PR TITLE
Add gas blast models

### PR DESCRIPTION
**Pull request by abdelhaqs**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [x] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

Fixing blast missing gas, related to l2_revenues https://github.com/duneanalytics/spellbook/issues/6596

### CODE REVIEW

**Comment by abdelhaqs** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jeff-dude** [comment]
hi @abdelhaqs thank you for dropping in to spellbook and looking to help contribute 🙏 

for gas fees spells, we have an active open PR to overhaul all EVM chains [here](https://github.com/duneanalytics/spellbook/pull/6562/files#diff-84fd74e735411ca87faef10d3098758c199031f65c0704780be5038d1daf4f2e) (link to blast, specifically). with that being the case, we will focus on finalizing there. we welcome your feedback on that PR, if you see anything to help us with!

note: if the macro is hard to follow, i'd suggest running `dbt compile` locally and looking at raw sql for the model 🫡
